projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -74,6 +74,20 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 		return parent::get_preferred_view( $screen, $fallback_global_preference );
 	}
 
+	/**
+	 * Point the Site Editor's `< Dashboard` link to wpcom home.
+	 *
+	 * Although this isn't strictly an admin menu item, it belongs here because it's part of
+	 * changing wp-admin links to their wp.com equivalents.
+	 *
+	 * @param  array $settings Editor settings.
+	 * @return array           Updated Editor settings.
+	 */
+	public function site_editor_dashboard_link( $settings ) {
+		$settings['__experimentalDashboardLink'] = 'https://wordpress.com/home/' . $this->domain;
+		return $settings;
+	}
```

Comment:
With this living in the masterbar module of the jetpack plugin, is this a change we would expect to effect non-wpcom sites?

Reply:
Oh nevermind, I see we are only adding this on the filters in the atomic and wpcom admin menu files.